src/import-data/protected-data-utils.ts

```diff
@@ -3,7 +3,7 @@ import { isGoogleWorkspace, WorkspaceWrapper } from 'src/libs/workspace-utils';
 
 // This method identifies whether an import source is considered protected data;
 // For now this means pfb imports from AnVIL or Biodata Catalyst.
-export const isProtectedSource = (url: string, filetype: string): boolean => {
+export const isProtectedSource = (url: string, filetype?: string): boolean => {
   if (!url) {
```

Comment:
Maybe this should check `filetype` too:
```
  if (!url || !filetype) {
```

Reply:
In theory, this should not have to check if `url` is defined because TS should require passing a string. However, all of these parameters are coming from query parameters with absolutely no validation of any kind. A future PR will add validation for the query parameters and clean this up.